bento_beacon/utils/censorship.py

```diff
@@ -2,6 +2,13 @@
 from .exceptions import APIException, InvalidQuery
 
 
+def no_results_censorship_message():
+    return (
+        "No results. Either none were found, or the query produced results numbering at "
```

Comment:
this might get a bit weird with translation - maybe there should be two info messages in order or something... although translating anything with a dynamic count threshold is going to be weird. maybe we need a separate field in `info` with the configured threshold, and set some flag in `info` if we show a message regarding the threshold.

Reply:
Ah, I wasn't going to propose translating this at all, since:
- it's the API json response, which is always English
- if a user-facing translation is needed somewhere (eg in bento_public), it can be translated there rather than in the API
- if bento public actually needs explicit mention of censorship, it needs to be somewhere other than hidden in an api response

Reply:
yes I dont think the response itself needs to be translated, but having a dynamic message (i.e., a changing threshold) means it's hard to translate on the bento_public end - if the threshold is set in the response and a flag can be set to show a special second info box with the threshold included, it would be easier to translate on that end